garak/generators/openai.py

```diff
@@ -190,6 +212,10 @@ def _call_model(
             if v is not None and k not in self.suppressed_params
         }
 
+        for k, v in self.custom_params.items():
+            if k not in self.suppressed_params:
+                create_args[k] = v
+
```

Comment:
This could work.

If I read it correctly, this exposes garak users to instabilities and changes in the OpenAI API. I think I'm mostly OK with that - we don't currently want to maintain a shim.

NB. garak defaults for `max_tokens` are too low to yield output when used for `max_completion_tokens` (aside from the `max_tokens` being only patchily supported), hence the new default